docs/en/developers/references/devex/cli.md

```diff
@@ -1,6 +1,6 @@
 # KubeVela CLI
 
-Learn about general configurations for `vela` command line tool. For the usage of the CLI, please check the KubeVela's [user documentation](../../README.md) instead.
+The fully specification of KubeVela's command line is automatically generated in this [directory](../../../cli).
```

Comment:
```suggestion
The fully specification of KubeVela's command line is automatically generated in the [cli directory](../../../cli).
```

Reply:
I think relative directory path won't work in website. Need to reference to https link.